core/node/libp2p/addrs.go

```diff
@@ -107,3 +131,40 @@ func ListenOn(addresses []string) interface{} {
 		}
 	}
 }
+
+func P2PForgeCertMgr(cfg config.ForgeClient) interface{} {
+	return func() (*p2pforge.P2PForgeCertMgr, error) {
+		storagePath, err := config.Path("", "p2p-forge-certs")
+		if err != nil {
+			return nil, err
+		}
+
+		certmagic.Default.Logger = logging.Logger("certmagic").Desugar()
```

Comment:
@aschmahmann thoughts on making this `p2p-forge/certmagic` ?

```suggestion
		certmagic.Default.Logger = logging.Logger("p2p-forge/certmagic").Desugar()
```